### PR TITLE
imx7ulp uboot 2021.04 fixes

### PR DESCRIPTION
**Pull request by vanmaegima**

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -201,6 +201,7 @@ UBOOT_CLASSES_imx7ulpea-ucom = "uboot-fitimage"
 BOOTSCR_LOAD_ADDR_imx7ulpea-ucom = "0x60800000"
 UBOOT_MACHINE_imx7ulpea-ucom = "mx7ulp_com_defconfig"
 UBOOT_CONFIG_imx7ulpea-ucom = ""
+PREFERRED_VERSION_u-boot-fio_imx7ulpea-ucom ?= "2021.04"
```
[the remark this answers is not in the document]
Comment (vanmaegima) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:204]: @angolini fixed the commit log.

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**